### DISCUSSION TITLE
How to remove the CoApp on Linux?

### DISCUSSION THREAD

**Discussion by seacat17** [aclap-dev/video-downloadhelper#33] [Help and Q&A] [answered]
### Your question

The question is simple. If I'll need to fully remove CoApp on Linux after tar installation - how do I do that?

**Comment by paulrouget** [comment, marked as answer]
First `./vdhcoapp uninstall`, this will remove the manifest files from the mozilla directory. Then just remove the vdhcoapp directory.
